.github/workflows/taiko-client--hive_test.yml

```diff
@@ -2,10 +2,7 @@ name: "Taiko Client Hive Tests"
 
 on:
   push:
-    branches: [main]
```

Comment:
revert it?

Reply:
It will be reverted after hive test passed.

Reply:
After hive test passed then merged to main